src/pet.js

```diff
@@ -57,6 +57,7 @@ Pet.prototype.feed = function () {
 };
 
 Pet.prototype.checkUp = function () {
+  // comment
```

Comment:
Avoid empty lines after a block of code starts, or before a block of code ends. The indentation alone should facilitate enough readability for these line breaks not to be necessary.

Reply:
Instead of really long `if / else if / else` chains, consider multiple `if` statements which you return inside of and a `return` statement afterwards that gets hit otherwise:

```js
if (name === 'Joe') {
  return 'Hello Joe';
}

if (name === 'Gaz') {
  return 'Hey Gaz!';
}

return 'Howdy.'
```